scripts/run_docker_build.sh

```diff
@@ -47,6 +47,6 @@ find conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt"
     | xargs -n1 cat | grep -v -e "^#" -e "^$" | \
     xargs -r yum install -y
 
-conda-build-all /conda-recipes --matrix-conditions "numpy >=1.9" "python >=2.7,<3|>=3.4"
+conda-build-all /conda-recipes --matrix-conditions "numpy >=1.8" "python >=2.7,<3|>=3.4"
```

Comment:
@pelson, how do we want to resolve this? These packages need NumPy 1.8.x, but I don't know that we want to build everything with 1.8.x. Will this be ok for that end goal or is there something more needed here?